src/yaml.ts

```diff
@@ -3,18 +3,22 @@
 // tslint:disable:variable-name
 
 import YAML from "yaml";
-import { CST } from "yaml/parse-cst.js";
+// eslint-disable-next-line import/extensions
+import { CST } from "yaml/parse-cst";
```

Comment:
Why should these imports be without extension?

Reply:
The old yaml use `exports: {'./': './'}`, can't be imported.